course_content/notebooks/numpy_intro.ipynb

```diff
@@ -781,29 +781,30 @@
       "\n",
       "For example, suppose we have :\n",
       "```python\n",
-      "A = np.array(\n",
-      "  [[1, 2, 3],\n",
-      "   [4, 5, 6],\n",
-      "   [7, 8, 9]])\n",
```

Comment:
I found this tight spacing made this cell hard to read, so I've stretched the spacing a bit.